src/Common/tests/System/PlatformDetection.cs

```diff
@@ -207,49 +207,72 @@ private static bool IsDistroAndVersion(string distroId, string versionId = null)
             return false;
         }
 
-        private static IdVersionPair ParseOsReleaseFile()
+        public static string GetDistroVersionString()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
+            {
+                return "";
+            }
+
+            DistroInfo v = ParseOsReleaseFile();
+
+            return "Distro=" + v.Id + " VersionId=" + v.VersionId + " Pretty=" + v.PrettyName + " Version=" + v.Version;
+        }
+
+        private static DistroInfo ParseOsReleaseFile()
```

Comment:
Now that I see this I wonder if it would make more sense to have these methods go into PlatformDetection.Unix file instead. For now I only moved code that added PInvokes to Unix native binaries, but I wonder if all of this Unix code should go there as well. No need to move it now, but we can log an issue to move it later if we think its valuable.